molgenis/molgenis-emx2
# Add services to staging area template

Closes issue https://github.com/molgenis/GCC/issues/810

### What are the main changes you did
- Add Services table to data model for staging areas
- Add Services table to demo data for staging areas

### How to test
- Create a new schema, upload the molgenis.csv data model, see Services
- Upload demo data to schema (zip CSVs), see Services table is filled
